api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -288,6 +288,50 @@ public void saveConcept_shouldForceSetFlagIfSetMembersExist() {
 		assertTrue(concept.getSet());
 	}
 	
+	/**
+	 * @see ConceptServiceImpl#saveDrug(Drug) 
+	 */
+	@Test
+	public void saveDrug_shouldPutGeneratedIdOntoReturnedDrug() {
+		executeDataSet("org/openmrs/api/include/ConceptServiceTest-initialConcepts.xml");
+		//New drug instance
+		Drug drug = new Drug();
+		drug.setConcept(conceptService.getConcept(1));
+		//Should have new ID generated for drug
```

Comment:
No meanings of this log in this line. Usually, we gave comments to complex lines or multiple conditions to make it easier.